src/assets/colors.json

```diff
@@ -0,0 +1,105 @@
+{
+    "reference": "https://medialab.github.io/iwanthue/",
+    "colors": [
```

Comment:
Seems like we'll need an arbitrarily large number of colors. There's a color space called L\*a\*b\* color which is meant to be that any change in any of the L, a, or b axes is always visually distinct. One such method, therefore, is to sample points at regular intervals in L\*a\*b\* and to convert the resulting colors to RGB to be stored in this file.

This removes the need to only have 100 colors references by another person and gives us the flexibility (if we so choose) to generate an arbitrary number of colors if we want to do it on-the-fly.

Reply:
I think that's a great sprint 2 goal, I generated these colors essentially on a whim and I'm not terribly pleased with their color range so I was wanting something more robust later. We can probably generate these colors as a function of the link ID of the dot itself which would probably be pretty clean